app/src/main/java/org/fossasia/openevent/general/auth/AuthHolder.kt

```diff
@@ -9,10 +9,7 @@ class AuthHolder(private val preference: Preference) {
 
     var token: String? = null
         get() {
-            if (field == null) {
-                field = preference.getString(TOKEN_KEY)
-            }
-
+            field = preference.getString(TOKEN_KEY)
```

Comment:
Why

Reply:
Are you sure about this? I am still not able to reproduce this error. Try deleting the data and starting the app again may be